AEPAnalytics/Tests/FunctionalTests/AnalyticsIDTests.swift

```diff
@@ -242,4 +242,92 @@ class AnalyticsIDTests : AnalyticsFunctionalTestBase {
         
         verifyIdentityChange(aid: nil, vid: "myvid")
     }
+    
+    func testHandleRequestResetEvent() {
+        let placesSharedState: [String: Any] = [
+            AnalyticsTestConstants.Places.EventDataKeys.CURRENT_POI : [
+                AnalyticsTestConstants.Places.EventDataKeys.REGION_ID : "myRegionId",
+                AnalyticsTestConstants.Places.EventDataKeys.REGION_NAME : "myRegionName"
+            ]
+        ]
+        let lifecycleSharedState: [String: Any] = [
+            AnalyticsTestConstants.Lifecycle.EventDataKeys.LIFECYCLE_CONTEXT_DATA : [
+                AnalyticsTestConstants.Lifecycle.EventDataKeys.OPERATING_SYSTEM : "mockOSName",
+                AnalyticsTestConstants.Lifecycle.EventDataKeys.APP_ID : "mockAppName",
+            ]
+        ]
+        
+        let dataStore = NamedCollectionDataStore(name: AnalyticsTestConstants.DATASTORE_NAME)
+        dataStore.set(key: AnalyticsTestConstants.DataStoreKeys.MOST_RECENT_HIT_TIMESTAMP, value: TimeInterval(100))
+        dataStore.set(key: AnalyticsTestConstants.DataStoreKeys.IGNORE_AID, value: true)
+        dataStore.set(key: AnalyticsTestConstants.DataStoreKeys.AID, value: "testaid")
+        dataStore.set(key: AnalyticsTestConstants.DataStoreKeys.VID, value: "testvid")
+        
+        mockNetworkService?.reset()
+        resetExtension()
+        
+        
+        simulateLifecycleState(data: lifecycleSharedState)
+        simulatePlacesState(data: placesSharedState)
+        dispatchDefaultConfigAndIdentityStates()
+        waitForProcessing()
+        
+        let analyticsProperties = analytics.analyticsProperties
```

Comment:
Let us not check internals in functional tests.

Reply:
You can either check it in following requests or use getTrackingIdentifier/getVisitorIdentifier APIs